website/docs/development/contributing.md

```diff
@@ -28,6 +28,30 @@ From the source tree, install Hydra in development mode with the following comma
 ```
 pip install -r requirements/dev.txt -e .
 ```
+
+### Website setup
+
+The website is built using [Docusaurus 2](https://v2.docusaurus.io/).
+
+**Install**
+
+```
+$ cd website
+$ yarn
+```
+
+**Local Development**
+
+```
+$ cd website
+$ yarn start
+```
+
+This command starts a local development server and open up a browser window. Most changes are reflected live without having to restart the server.
+
+For more details, refer [here](https://github.com/facebookresearch/hydra/blob/master/website/README.md).
+
+
```

Comment:
```suggestion

### Website setup

The website is built using [Docusaurus 2](https://v2.docusaurus.io/).  
Run the following commands from the `website` directory.

#### Install
```
$ yarn
```

#### Local Development
```
$ yarn start
```

This command starts a local development server and open up a browser window. Most changes are reflected live without having to restart the server.

For more details, refer [here](https://github.com/facebookresearch/hydra/blob/master/website/README.md).


```

Reply:
There is no real change - the size is the same.

Reply:
Fixed in https://github.com/facebookresearch/hydra/pull/819/commits/6c6ad19a72e353701339b49676da659e2f4cb31d

Reply:
Also, at this point the contributing guide is getting very large.
I think it's time to split it into smaller sections. this can be a followup diff.

Reply:
what do you mean "no real change"?
you didn't apply the suggestion.

Reply:
damn github.
let me just edit it.

Reply:
ahh isnt `cd website` necessary since `package.json` is in `website` ?